### ISSUE TITLE
[Twig_Error_Syntax] Unknown "preload" function.

### ISSUE THREAD

**Issue by medooch** [symfony/symfony#22672]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Symfony version  | 3.3

I try to test the most important components added in version 3.3.
I will start with this bug report.

**Comment by robfrawley** [comment]
Under what circumstances does this error occur for you? Perhaps you can provide additional information? I'd suggest following the guidelines in the [submitting a bug report](http://symfony.com/doc/current/contributing/code/bugs.html) documentation and fork the [standard edition](https://github.com/symfony/symfony-standard/) to create a reproduction.

I can confirm that this bug doesn't show up when working with the base Symfony `3.3.x-dev` release using the Symfony Installer

```bash
symfony new symfony-test-3.3.x-dev 3.3
cd symfony-test-3.3.x-dev
composer update
php bin/console server:start
```

**Comment by robfrawley** [comment]
Possibly of note, the `preload` function was moved from the `AssetExtension` to the `WebLinkExtension` within the `symfony/twig-bridge` package.

https://github.com/symfony/symfony/pull/22273/files#diff-60f9f82f1168a6e3c5ea6e8f57fd0b12R39.

**Comment by chalasr** [comment]
This should have been fixed by https://github.com/symfony/symfony/pull/22639

**Comment by xabbuh** [comment]
Closing here as it should have been fixed by #22639 as @chalasr said. @medooch Please leave a comment with more information on how to reproduce if you still encounter the issue and we can reopen.
